src/components/Button/index.tsx

```diff
@@ -54,10 +54,10 @@ const Button = React.forwardRef<HTMLButtonElement, Props>(function Button(
     ref,
 ) {
     const className = classNames(
-        'reactist_button',
-        variant ? `reactist_button--${variant}` : null,
-        size !== 'default' ? `reactist_button--${size}` : null,
-        { 'reactist_button--loading': loading },
+        styles['reactist_button'],
```

Comment:
With a css modules approach in place I'd rename all this classes to get away from a BEM-style naming scheme to one more unconcerned with name clashes, since css modules ensure these are local and not global.

- `.reactist_button` can become `.button` or `.container`
- `.reactist_button--xyz` can become `.xyz` for any "xyz" that we may have.
- camelCase should be used for any class name that needs more than one word in it. The js code accessing styles should not need the square-brackets notation to access any of the attributes of `styles`.

Reply:
Can do this, but moving out of `reactist_button` classes might be harder than I thought coz of [styles like this](https://github.com/Doist/twist-web/pull/2461/files#diff-2a77a1500ab87d4918863f5f4d065bafR337). 

Anyways can only land this after the theming PR drops as that's the only way to move such styles to within reactist.